X/xorriso/build_tarballs.jl

```diff
@@ -0,0 +1,67 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "xorriso"
+version = v"1.5.5"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("https://github.com/gnu-mirror-unofficial/xorriso/raw/refs/heads/master/xorriso-1.5.5.tar.gz", "89a78b902ded443c3e4b31b3ba586ccbf06a447836d37f1082dfb1e429952217")
```

Comment:
```suggestion
    ArchiveSource("https://github.com/gnu-mirror-unofficial/xorriso/raw/refs/heads/master/xorriso-$(version).tar.gz", "89a78b902ded443c3e4b31b3ba586ccbf06a447836d37f1082dfb1e429952217")
```

Reply:
Also, this doesn't look like a very official link.  Why not using https://ftp.gnu.org/gnu/xorriso/ instead?

Reply:
Also, version 1.5.6.pl02 looks like it's the current version.

Reply:
I updated the source with the official link